apps/passport-server/test/frogcryptodb.spec.ts

```diff
@@ -126,4 +130,25 @@ describe("database reads and writes for frogcrypto features", function () {
     const possibleFrogIds = await getPossibleFrogIds(db);
     expect(possibleFrogIds).to.deep.eq([1, 2, 3, 7, 8]);
   });
+  
+  step("insert feeds", async function () {
+    await upsertFeedData(db, testFeeds);
+
+    const allFeeds = await getFeedData(db);
+    expect(allFeeds.length).to.eq(testFeeds.length);
+  });
+
+  step("update feeds", async function () {
+    const mutatedFeed = JSON.parse(
+      JSON.stringify(testFeeds[3])
+    ) as FrogCryptoDbFeedData;
+    mutatedFeed.feed.private = false;
+    await upsertFeedData(db, [mutatedFeed]);
+
+    const allFeeds = await getFeedData(db);
+    expect(allFeeds.length).to.eq(testFeeds.length);
+    expect(
+      allFeeds.find((feed) => feed.id === mutatedFeed.uuid)?.private
+    ).to.eq(false);
+  });
```

Comment:
maybe a few more checks here would be nice, just to be sure that upserting feeds does the right thing